Specs/Scene/LabelCollectionSpec.js

```diff
@@ -585,6 +585,7 @@ defineSuite([
         expect(scene).toRender([0, 0, 0, 255]);
 
         label.scale = 2.0;
+        scene.render();
```

Comment:
How is test coverage?  Are we sure there is enough new tests to cover the new code paths?  Or do existing tests cover the new code paths in a reasonable way?

Reply:
I think test coverage is pretty good.  The underlying SDF rendering logic isn't so different now than what was in there before so all the rendering tests should remain the same.  There were also some good tests in there before regarding the number of glyphs and when new ones were added based on property changes that changed due to the new SDF logic, so all of those tests have been updated to reflect the new logic.  So for example previously we had tests for changing the outline width resulting in a new glyph being created, and now that test has changed to make sure a new glyph is NOT created.  If there are other places though that you think need addressed let me know though and I'll take a look.